src/hlstrafe.cpp

```diff
@@ -685,6 +685,12 @@ namespace HLStrafe
 			theta = right ? -theta : theta;
 
 			double yaw = vel_yaw - phi + theta;
+
+			if (curState.ConstantYawSpeed) {
+				auto max_yaw_delta = curState.ConstantYawSpeedValue * vars.Frametime;
+				yaw = (right ? curState.LastYaw - max_yaw_delta : curState.LastYaw + max_yaw_delta) * M_DEG2RAD;
```

Comment:
so player.Viewangles[1] cannot be used here? It's already modified by something before this code?

Reply:
Hmmm I didn't think of that.